superset-embedded-sdk/src/index.ts

```diff
@@ -154,6 +154,7 @@ export async function embedDashboard({
       });
 
       iframe.src = `${supersetDomain}/embedded/${id}${dashboardConfig}${filterConfigUrlParams}`;
+      iframe.title = 'Embedded Dashboard';
```

Comment:
This works in a pinch, but does the dashboard config (or something else) happen to provide the actual title of the dashboard? That would be the ideal thing to put here.

Reply:
It seems the sdk would expose the option to set this `iframeTitle`, and fall back on some default like this one "Embedded Dashboard".  Note that for extra points you could i18n that string as well.

Reply:
Moved it to the config. Didn't provide a fallback for now because I'd probably butcher the i18n. Would a fallback be required for this, or okay without?

Reply:
Ok without AFAIC, the i18n is easy though:

```js
import { t } from '@superset-ui/core';

translatableString = t("Embedded Dashboard");
```

What happens then is the i18n framework scans through the code base and inventorize all those strings on the backend and frontend, where other people can then translate then in various languages.